packages/content-req-app/src/app/pr-list/pr-list.component.ts

```diff
@@ -1,35 +1,18 @@
-import {Component, OnInit} from '@angular/core';
-import {
-  Listener,
-  Message,
-  MessagingService,
-  Permission,
-  PluginComponent,
-  PluginConfiguration,
-  PluginContext,
-  TextMessage,
-  TopicPublisher
-} from '@fundamental-ngx/app-shell';
+import {Component, OnDestroy, OnInit} from '@angular/core';
+import {Message, MessagingService, TextMessage, TOPIC_THEME_CHANGE, TopicSubscriber} from '@fundamental-ngx/app-shell';
 
 
-/**
- * Should we have interface or crete custom decorator
- *
- * ```ts
- * @PluginComponent({
- *   configuration: PrListPluginConfiguration
- * })
- * ```
- * but then I still want to initialize ?
- *
- */
 @Component({
   selector: 'app-pr-list',
   templateUrl: './pr-list.component.html',
 })
-export class PrListComponent implements OnInit, PluginComponent {
+export class PrListComponent implements OnInit, OnDestroy {
   tableRows: any[];
-  private messaging: MessagingService;
+  subscriber: TopicSubscriber<Message>;
+
+  constructor(private messaging: MessagingService) {
+    this.messaging = messaging;
```

Comment:
access modifier will do it for you

Reply:
Yeah, this is left over,  as I had exising member that I was not settings from contructor. we dont need this anymore